.devops/templates/tools.yml

```diff
@@ -2,7 +2,8 @@
 steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: '12.x'
+      versionSpec: '14.18.1'
```

Comment:
bumps agent host machine `node` version to version `14.18.1` to align with master (#25946) and to make it compatible with the `storywright` tool.